package.json

```diff
@@ -58,11 +58,11 @@
     "ember-link-action": "0.0.35",
     "ember-load-initializers": "^0.5.1",
     "ember-metrics": "0.6.3",
-    "ember-osf": "git+https://github.com/CenterForOpenScience/ember-osf.git#db96ee083e439ba26b4f62320a4482967b11d1aa",
+    "ember-osf": "git+https://github.com/CenterForOpenScience/ember-osf.git#873b1e360d2762fb8d7c1cb3fada89c4b60b658f",
     "ember-page-title": "3.0.6",
     "ember-power-select": "1.0.0-beta.7",
     "ember-read-more": "0.0.13",
-    "ember-resolver": "2.1.0",
+    "ember-resolver": "^2.0.3",
```

Comment:
Let's leave this pinned at 2.1.0 for now, at least until yarn goes through, to avoid merge conflicts.